### DISCUSSION TITLE
【论坛转发】充值流程+登录流程

### DISCUSSION THREAD

**Discussion by kebiao** [kbengine/kbengine#1370] [General]
充值流程（无回调地址）：
![image](https://user-images.githubusercontent.com/2352650/107493034-1c751b00-6bc8-11eb-8472-eea3eafad7a4.png)

充值流程（有回调地址）：
![image](https://user-images.githubusercontent.com/2352650/107492914-ef286d00-6bc7-11eb-8424-f8765347678e.png)

IOS支付流程：
![image](https://user-images.githubusercontent.com/2352650/107493178-4b8b8c80-6bc8-11eb-881a-79b2b0a7cfb1.png)

登录流程：
![image](https://user-images.githubusercontent.com/2352650/107493247-5cd49900-6bc8-11eb-9050-9b1b20446442.png)

**Comment by kebiao** [comment]
reply: 小菜鸟, date: 2019-10-24 18:00:03, 
战略性 Mark , 感谢分享 ！

reply: kylinok, date: 2019-10-24 23:04:19, 
记录一下

reply: lehaha888, date: 2019-10-25 11:02:33, 
战术性马克

reply: xiaobao4106, date: 2020-09-17 13:53:16, 
好文章，收藏

reply: 柯标, date: 2019-10-24 21:31:18, 
同一个号？

reply: FreeDomShell, date: 2019-10-24 22:54:55, 
[quote][size=2][url=forum.php?mod=redirect&goto=findpost&pid=41904&ptid=8539][color=#999999]柯标 发表于 2019-10-24 21:31[/color][/url][/size]

同一个号？[/quote]

同一账号，手机端经常出现

reply: 柯标, date: 2019-10-25 09:19:43, 
PC试试， demo试试

**Reply by datemore111** [reply]
### reply: 小菜鸟, date: 2019-10-24 18:00:03, 
战略性 Mark , 感谢分享 ！

### reply: kylinok, date: 2019-10-24 23:04:19, 
记录一下

### reply: lehaha888, date: 2019-10-25 11:02:33, 
战术性马克

### reply: xiaobao4106, date: 2020-09-17 13:53:16, 
好文章，收藏

### reply: 柯标, date: 2019-10-24 21:31:18, 
同一个号？

### reply: FreeDomShell, date: 2019-10-24 22:54:55, 
引用：柯标 发表于 2019-10-24 21:31

同一个号？

---




同一账号，手机端经常出现

### reply: 柯标, date: 2019-10-25 09:19:43, 
PC试试， demo试试

**Reply by goddissme** [reply]
mark
